corehq/apps/data_analytics/tasks.py

```diff
@@ -23,11 +25,11 @@
 def build_last_month_MALT():
     last_month = last_month_dict()
     domains = Domain.get_all_names()
-    grouped_malt_tasks = update_current_MALT_for_domains.chunks(
-        zip([last_month], domains), 1000
-    ).group()
+    tasks = []
+    for chunk in chunked(domains, 1000):
+        tasks.append(update_current_MALT_for_domains.s(last_month, list(chunk)))
 
-    group_result = grouped_malt_tasks.apply_async()
+    group_result = group(tasks).apply_async()
```

Comment:
I think you don't need the `group` here at all, especially since we aren't really using any of it's features. I think you should be able to just do `tasks.append(update_current_MALT_for_domains.delay(last_month, list(chunk)))` and then iterate over `tasks` in line 35 below.

Reply:
This crossed my mind. My only hesitation was that it would be nice to use a group once we upgrade so that we can just wait on the entire group. Functionally, it is the exact same though. Updated 👍